prost-types/src/any.rs

```diff
@@ -20,16 +20,13 @@ impl Any {
     {
         let expected_type_url = M::type_url();
 
-        match (
+        if let (Some(expected), Some(actual)) = (
```

Comment:
Could this be further simplified?
```suggestion
        if TypeUrl::new(&expected_type_url) == TypeUrl::new(&self.type_url) {
```